.github/templates/windows_ci_workflow.yml.in

```diff
@@ -99,12 +100,18 @@ jobs:
           if-no-files-found: error
           path:
             test/**/*.xml
-  # TODO: Make this into a composite step
+
+  # this is a separate step from test because the log files from test are too
+  # long: basically, GitHub tries to render all of the log files when you click
+  # through an action causing extreme slowdown on actions that contain too many
+  # logs (like test); we can always move it back to the other one, but it
+  # doesn't create the best experience
   render_test_results:
     if: always()
     needs:
       - test
     runs-on: ubuntu-18.04
+    # TODO: Make this into a composite step
```

Comment:
not sure why this comment was moved from job level to step level.

Reply:
1. to separate it from the new comment
2. because it doesn't apply to the job as a whole, it only really applies to the sequence of `steps`

Reply:
Ah.. sounds good. it wasnt' clear what "this" meant LOL